docs/components/datetimepicker.md

```diff
@@ -0,0 +1,409 @@
+</div>
+
+```html
+<KDateTimePicker
+  v-model="currentValue"
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a time frame"
+  mode="relative"
+  :timePeriods=[
+    {
+      section: "Last",
+      values: [
+        { key: "15m", prefix: "Last", timeframeText: "15 minutes", start: function() {}, end: function() {} },
+        { key: "12h", prefix: "Last", timeframeText: "12 hours", start: function() {}, end: function() {} },
+        { key: "24h", prefix: "Last", timeframeText: "24 hours", start: function() {}, end: function() {} },
+        ...
+      ]
+    },
+    {
+      section: "Current",
+      values: [
+        { key: "current_week", prefix: "Current", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "current_month", prefix: "Current", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+    {
+      section: "Previous",
+      values: [
+        { key: "previous_week", prefix: "Previous", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "previous_month", prefix: "Previous", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+  ]
+/>
+```
+
+## Props
+
+### v-model
+
+#### Single date time picker `v-model`
+
+A single date or time picker instance will emit a Javascript Date object as a `string`.
+
+```js
+"2022-09-06T19:45:54.406Z"  // ISO 8601 Date string
+```
+
+#### Range date time picker `v-model`
+
+An instance that displays a date range or a series of relative time frames will emit an `object` containing the following values:
+
+```js
+{
+  "start": "2022-09-06T19:45:54.406Z",  // ISO 8601 Date string
+  "end": "2022-09-07T19:45:54.406Z", // ISO 8601 Date string
+  "timePeriodsKey": "24h" // or "" if not applicable
+}
+```
+
+### maxDate
+
+`Number` which blocks out days/times that occur **after** the given timestamp. `maxDate` gets passed down to the calendar component, and does not apply to relative time frames.
+
+**default**: `null`
+
+### minDate
+
+`Number` which blocks out days/times that occur **before** the given timestamp. `minDate` gets passed down to the calendar component, and does not apply to relative time frames.
+
+**default**: `null`
+
+### mode
+
+`String` parameter which accepts the following values: `date`, `time`, `dateTime` and `relative`. The latter is meant to denote an datetime picker instance which only contains relative time frames.
+
+### placeholder
+
+`String` containing the default message displayed in the date time picker input. Clearing any selected values will revert the input to display this placeholder message.
+
+**default**: `Select a time range`
+
+### range
+
+`Boolean` which determines whether the calendar allows selection of a **single** date or time, as opposed to a **range** of start and end values. This parameter, along with the `mode` setting, gets passed down to the calendar component.
```

Comment:
We don't need to inform the user how the underlying external package component works.

```suggestion
`Boolean` which determines whether the calendar allows selection of a **single** date or time, as opposed to a **range** of start and end values.
```